root/components/custom-searchbar.js

```diff
@@ -9,30 +9,38 @@
     { id: 'vegan', name: 'Vegan' },
     { id: 'vegetarian', name: 'Vegetarian' },
   ];
-
-  const checkboxContainer = document.createElement('form');
+  const checkboxContainer1 = document.createElement('div');
+  checkboxContainer1.classList.add('checkboxes1');
+  const checkboxContainer = document.createElement('div');
   checkboxContainer.id = 'checkboxes';
   checkboxContainer.classList.add('checkboxes');
 
   tagProperties.forEach((tag) => {
-    const tagCheckbox = document.createElement('label');
-    tagCheckbox.classList.add('container');
-    tagCheckbox.innerText = tag.name;
+    const func = document.createElement('div');
+    func.classList.add('checked');
 
     const inp = document.createElement('input');
     inp.id = tag.id;
     inp.name = tag.name;
     inp.type = 'checkbox';
+    inp.classList.add('checkbox');
+
+    const tagCheckbox = document.createElement('label');
+    tagCheckbox.classList.add('container');
+    tagCheckbox.innerText = tag.name;
 
-    const inpspan = document.createElement('span');
-    inpspan.classList.add('checkmark');
+    
```

Comment:
Codacy has a fix for the issue: [Trailing spaces not allowed.](https://app.codacy.com/gh/cse110-fa21-group34/rocketrecipes/pullRequest?prid=8525839)
```suggestion

```